girder/models/model_base.py

```diff
@@ -892,7 +1023,7 @@ def getFullAccessList(self, doc):
 
         return acList
 
-    def setUserAccess(self, doc, user, level, save=False):
+    def setUserAccess(self, doc, user, level, save=False, flags=None, currentUser=None):
```

Comment:
by the same logic that `setPublicFlags` has a `force` param, this method should too